### PR TITLE
Updating to work with new GitHub Projects

### LINKED ISSUE #10: Update to work with Projects V2

### PR DESCRIPTION
**Pull request by evanjamesjackson**
Resolves #10.

The Action will now work with GitHub's "V2" Projects. 

Additionally, running the Action on the `pull_request_review: types: submitted` event will reassign the issue to the original PR creator, and move it to the "In Progress" status, if the reviewer has requested changes.

### CODE REVIEW

**Review by nick-compassaq — CHANGES_REQUESTED** [review]
requesting changes

**Review by nick-compassaq — CHANGES_REQUESTED** [review]
requesting more changes

File: src/main.ts [hunk 1]
```diff
@@ -1,169 +1,293 @@
 import * as core from '@actions/core'
```
Comment (nick-compassaq) [inline src/main.ts:1]: I'm gonna give it to you straight, this 300 line file stuffed with 50 line functions is kind of scary. It probably looks clearer in an IDE when you can ctrl+click around, but I feel like this'd just benefit from at least breaking up the functions into different files. I'd even make a case for trying to find a more object-oriented approach to doing this, but that's a way bigger refactor, so I'm perfectly fine to settle for just breaking this up a little bit.

**Review by nick-compassaq — CHANGES_REQUESTED** [review]
As we've verified, this definitely works but that `main.ts` is kind of frightening

File: src/main.ts (hunk 1 above)
Comment (evanjamesjackson) [reply src/main.ts:1]: How's that?

**Review by nick-compassaq — APPROVED** [review]
I can live with that!

**Pull request merged**